src/x86/rfxcodec_encode_dwt_shift_x86_sse2.asm

```diff
@@ -21,6 +22,29 @@
 
 %ifidn __OUTPUT_FORMAT__,elf
 section .note.GNU-stack noalloc noexec nowrite progbits
+%ifdef PIC
+section .text
+extern _GLOBAL_OFFSET_TABLE_
+..@get_GOT:
+	mov ebx,[esp]
+	ret
+%define lsym(name) ebx + name wrt ..gotoff
+%macro get_GOT 0
+	call ..@get_GOT
+	add ebx,_GLOBAL_OFFSET_TABLE_+$$-..@get_GOT wrt ..gotpc
```

Comment:
I wanted to use more tabs and spaces, but the current coding style didn’t quite match. But yes, I see there are _some_ spaces in it, so I’ll fix this later.